tests/e2e/core-tests/orders_01_single_test.js

```diff
@@ -1,7 +1,7 @@
 const helpers = require('../pages/helpers');
 Feature('Single order Functionality');
-Scenario('Customer single Order', ({ I }) => {
-    I.loginAsCustomer();
+Scenario('Customer single Order', ({ I,loginAs }) => {
+    loginAs('customer');
```

Comment:
Maintain upper and lower case for the user role because the user is case sensitive.